x-pack/plugins/security_solution/public/cloud_security_posture/components/csp_details/misconfiguration_findings_details_table.tsx

```diff
@@ -202,7 +225,14 @@ export const MisconfigurationFindingsDetailsTable = memo(
             <EuiIcon type={'popout'} />
           </SecuritySolutionLinkAnchor>
           <EuiSpacer size="xl" />
-          <DistributionBar stats={getFindingsStats(passedFindings, failedFindings)} />
+          <DistributionBar
+            stats={getFindingsStats(
```

Comment:
We should move the `getFindingsStats` function to a const,  this function will cause unnecessary re-renders since it will execute render.

```
const stats = getFindingsStats(....)
```